### PR TITLE
Add element dupe advice to FAQ

### PR DESCRIPTION
**Pull request by bootstraponline**
#104

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -142,3 +142,10 @@ Refer to the [PSPDFKit blog post for more details.](https://pspdfkit.com/blog/20
 ```swift
 UIApplication.sharedApplication().keyWindow?.layer.speed = 100
 ```
+
+**How do I match an element when it's duplicated in the app?**
+
+EarlGrey requires all matchers return exactly one element.
+This is difficult to do when an element is duplicated (same label/class/location).
+
+Try adding `grey_interactable()` or `grey_sufficientlyVisible` to the `grey_allOfMatchers()`.
```
Comment (tirodkar) [inline docs/faq.md]: `grey_sufficientlyVisible()`. Also, user grey_allOf()... for Objective-C.

**Comment by bootstraponline** [comment]
Updated.

File: docs/faq.md [hunk 2]
```diff
@@ -142,3 +142,11 @@ Refer to the [PSPDFKit blog post for more details.](https://pspdfkit.com/blog/20
 ```swift
 UIApplication.sharedApplication().keyWindow?.layer.speed = 100
 ```
+
+**How do I match an element when it's duplicated in the app?**
+
+EarlGrey requires all matchers return exactly one element.
+This is difficult to do when an element is duplicated (same label/class/location).
+
+First use `grey_allOfMatchers(...)` (Swift) or `grey_allOf(...)` (Objective C) then
```
Comment (tirodkar) [inline docs/faq.md]: I'd rephrase this to:

"We recommend combining the matchers as suggested [here](https://github.com/bootstraponline/EarlGrey/blob/0715a8b5c8b23861862246f28aab299294eb679c/docs/api.md#earlgrey-matchers) and then adding `grey_interactable()` or `grey_sufficientlyVisible()`."

**Comment by bootstraponline** [comment]
Updated.

**Comment by khandpur** [comment]
LGTM
